pkg/annotations/ingress_annotations_test.go

```diff
@@ -0,0 +1,37 @@
+package annotations
```

Comment:
We need a CMAKE target to run these tests in CI?

Reply:
I'll make sure to add a target for running tests.

This is work-in-progress BTW - still working on the PR...